domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleResponseAcceptation.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.domain.models.sampledata;
+
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.FormaliseOption;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation.ResponseAcceptationBuilder;
+import uk.gov.hmcts.cmc.domain.models.sampledata.response.SampleCourtDetermination;
+import uk.gov.hmcts.cmc.domain.models.sampledata.response.SamplePaymentIntention;
+
+import java.math.BigDecimal;
+
+public class SampleResponseAcceptation {
+
+    private SampleResponseAcceptation(){}
+
+    public static ResponseAcceptationBuilder builder() {
```

Comment:
shouldnt this be a private and should be used only this class other static methods.

Reply:
no ..not necessarily..I think if the builder() is accessible then it allows for more flexibility. Imagine if all our domain clas had private builder()...

Reply:
What I meant was in the SampleResponseAcceptation we are having builder() method that gives builder of ResponseAcceptationBuilder.